Public/Src/Tools/BuildXL.Tools.CredentialProvider/BuildXL.Tools.CredentialProvider.csproj

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props" Condition="Exists('$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props')" />
+  <PropertyGroup>
+    <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
+    <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
+    <ProjectGuid>{A7C3C1F0-7800-4D4D-BBCB-D69483A1E2E3}</ProjectGuid>
+    <OutputType>Exe</OutputType>
+    <RootNamespace>CredentialProviderBuildXL</RootNamespace>
+    <AssemblyName>CredentialProviderBuildXL</AssemblyName>
+    <TargetFrameworkVersion>v4.7.2</TargetFrameworkVersion>

```

Comment:
We are generally moving away from Full Framework, so there is no reason not to use .NET Core here as well.